spec/features/reply_spec.rb

```diff
@@ -0,0 +1,27 @@
+feature 'Adding tags' do
+
+  def sign_in(user)
+    visit'/sessions/new'
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.